invisible_cities/detsim/light_tables.py

```diff
@@ -0,0 +1,103 @@
+import numpy  as np
+import pandas as pd
+import warnings
+from typing import Callable
+from typing import Optional
+
+from functools import partial
+from .. core                import system_of_units as units
+from .. core.core_functions import binedges_from_bincenters
+
+from .. io.dst_io  import load_dst
+
+
+def read_lt(fname, group_name, el_gap=None, active_r=None):
+    """ A helper function to extract dataframes and configuration infro from files.
+    Inputs
+        fname      : string of file name
+        group_name : group name under which table is saved
+        el_gap     : optional set of different el_gap
```

Comment:
review docstrings; ie should be 
```
param : type 
        explanation
```
`el_gap` and `active_r`  explanation should probably be  'optionally set a different el_gap' or similar

In a returns description `el_gap` is repeated